recipes/metview/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e # Abort on error.
+
+export PING_SLEEP=30s
+export WORKDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+export BUILD_OUTPUT=$WORKDIR/build.out
+
+touch $BUILD_OUTPUT
+
+dump_output() {
+    echo Tailing the last 500 lines of output:
+    tail -50000 $BUILD_OUTPUT
+}
+error_handler() {
+    echo ERROR: An error was encountered with the build.
+    dump_output
+    exit 1
+}
+
+# If an error occurs, run our error handler to output a tail of the build.
+trap 'error_handler' ERR
+
+# Set up a repeating loop to send some output to Travis.
+bash -c "while true; do echo \$(date) - building ...; sleep $PING_SLEEP; done" &
```

Comment:
Looks like this is useful for "debugging" a build, but consider removing it from the final recipe

Reply:
No, the point is to limit the amount of log output. We ran into some trouble a while back (2017 according to git history) because we were hitting the Travis log line limit when building some ECMWF packages. Perhaps something has changed since then - if you can confirm it's not needed then I'll delete it.

Reply:
Ah but we no longer use Travis on conda-forge do we! This work around is not necessary on Azure - I'll remove it.